pyproject.toml

```diff
@@ -97,33 +97,33 @@ pep440 = "commitizen.version_schemes:Pep440"
 semver = "commitizen.version_schemes:SemVer"
 
 [tool.coverage]
-    [tool.coverage.report]
-    show_missing = true
-    exclude_lines = [
-        # Have to re-enable the standard pragma
-        'pragma: no cover',
-
-        # Don't complain about missing debug-only code:
-        'def __repr__',
-        'if self\.debug',
-
-        # Don't complain if tests don't hit defensive assertion code:
-        'raise AssertionError',
-        'raise NotImplementedError',
-
-        # Don't complain if non-runnable code isn't run:
-        'if 0:',
-        'if __name__ == .__main__.:',
-        'if TYPE_CHECKING:',
-    ]
-    omit = [
-        'env/*',
-        'venv/*',
-        '.venv/*',
-        '*/virtualenv/*',
-        '*/virtualenvs/*',
-        '*/tests/*'
-    ]
+[tool.coverage.report]
```

Comment:
Are you using some toml linter or autoformatter?

Reply:
Wow - yes, I just switched to a new extension due to the old one being deprecated. Did not expect it to apply formatting. I'll get this cleaned up. My apologies!

Reply:
np. could you please share what's the tool you're using? just curious haha